tests/scale_test/utils.py

```diff
@@ -0,0 +1,238 @@
+import json
+import uuid
+import requests
+import click
+
+
+def send_notification(lira_url, auth_token, notification):
+    """Send a notification to a given Lira.
+
+    Args:
+        lira_url (str): A typical Lira url, e.g. https://pipelines.dev.data.humancellatlas.org/
+        auth_token (str): The Lira auth_token.
+        notification (dict): A dict of notification content.
+
+    Returns:
+        requests.Response: The response object returned by Lira.
+    """
+    response = requests.post(url=lira_url + 'notifications', json=notification, params={'auth': auth_token})
```

Comment:
I think this should use the `harmonize_url` function here as well for the lira_url

Reply:
Yep, nice catch!